ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -3,12 +3,14 @@ import Outputs from '../../../../../resources/outputs';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
+import { NetMatches } from '../../../../../types/net_matches';
 import { TriggerSet } from '../../../../../types/trigger';
 
 export interface Data extends RaidbossData {
   partnersSpreadCounter: number;
   storedPartnersSpread?: 'partners' | 'spread';
   beat?: 1 | 2 | 3;
+  tankLaserCollect: NetMatches['HeadMarker'][];
```

Comment:
More a suggestion/for future reference, but you can also just make this a `string[]` and push `matches.target` rather than the entire matches object (and no need for importing the type then).  If not using anything other than the target field, it's not necessary to store the entire regex capture object on `data`.